sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_complete_order_discrepancy_inference.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import datetime
+import os
+import uuid
+
+
+from azure.identity import DefaultAzureCredential
+from azure.healthinsights.radiologyinsights import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
```

Comment:
Nit: I would move these imports into the main sample definition. comment applies across samples.